Symfony/CS/ConfigInterface.php

```diff
@@ -105,4 +105,20 @@ public function usingCache();
      * @return bool
      */
     public function usingLinter();
+
+    /**
+     * Sets the path to the cache file.
+     *
+     * @param string $cacheFile The path to the cache file
+     *
+     * @return ConfigInterface The same instance
```

Comment:
Don't understand you gays here.
`addCustomFixer` is chainable therefore it should be documented in `ConfigInterface`.
Why? one day it was not chainable, we decided to make it chainable and won't change the interface to not introduce BC break change. Now you are PRing into 2.0, therefore we can.

`setCacheFile` should be chainable too.
**All** other setters are chainable.